client/src/components/QA/AnswersList.jsx

```diff
@@ -24,14 +25,29 @@ const AnswersList = ({ questionId }) => {
   useEffect(() => {
     axios.get(`/api/qa/questions/${question}/answers`, { params: { count: limit } })
       .then(({ data }) => {
-        // console.log('results from answerslist.jsx', data.results);
         setAnswers(data.results);
       })
       .catch((err) => {
         throw err;
       });
   }, [limit]);
 
+  const showMore = (ans) => {
+    if (ans.length === 0) {
+      return (<p> There are no answers. Add a new answer</p>);
+    }
+    if (limit !== ans.length) {
+      return '';
+    }
+    if (ans.length > 0) {
+      return (
+        <LoadBtn onClick={() => { setLimit((prevState) => prevState + 2); }}>
+          Load More Answers
+        </LoadBtn>
+      );
+    }
+  };
+
```

Comment:
The only problem is when there are an even number of answers, it will show the show more button and when you click it it will disappear, but I'm not sure how to fix it using the way you did it!